### PR TITLE
Embedded faq

### PR DESCRIPTION
**Pull request by strump**
Added Zola template to generate FAQ simmilar to https://github.com/organicmaps/organicmaps/blob/master/data/faq.html

See [`/faq/embedded-faq?lang=en`](https://embedded-faq.organicmaps.pages.dev/faq/embedded-faq/?lang=en), or [`/faq/embedded-faq?lang=pl`](https://embedded-faq.organicmaps.pages.dev/faq/embedded-faq/?lang=pl), or [`/faq/embedded-faq?lang=ru`](https://embedded-faq.organicmaps.pages.dev/faq/embedded-faq/?lang=ru), etc.

<s>**Problem 1:** there are no questions "How to create a route and start navigation" and "I can't hear voice instructions" in https://organicmaps.app/faq/</s>

Added two more questions to `master` branch and to embedded FAQ page.

<s>**Problem 2:** New strings "For more information please visit our web-site" and "Go to Top" should be translated in config.toml but I just put HTML code with translations into `embedded-faq.html`.</s>

Added two strings `more-info-om` and `go-to-top` to translations list.

FAQ looks and works good in Android and iOS:

### CODE REVIEW

**Comment by biodranik** [comment]
There is a non-matching p tag:

выява

**Comment by strump** [comment]
@biodranik this promes is caused by `<ul>` tag within `<p>` tag. So I removed outer `<p>` tag. Anyway Markdown-to-HTML convertor adds those tags automatically

File: templates/faq/embedded-faq.html [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+        For more information please visit our web-site
+        <a href="https://organicmaps.app/">organicmaps.app</a>
+    </h4>
+    <h4 lang="ru">
+        Больше информации на нашем сайте
+        <a href="https://organicmaps.app/ru/">organicmaps.app</a>
+    </h4>
+    <h4 lang="de">
+        Für weitere Informationen besuchen Sie bitte unsere Website
+        <a href="https://organicmaps.app/">organicmaps.app</a>
```
Comment (biodranik) [inline templates/faq/embedded-faq.html:175]:
```suggestion
        <a href="https://organicmaps.app/de/">organicmaps.app/de/</a>
```

File: templates/faq/embedded-faq.html [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+    {% endfor %}
+    </ul>
+
+    <h4 lang="en">
+        For more information please visit our web-site
+        <a href="https://organicmaps.app/">organicmaps.app</a>
+    </h4>
+    <h4 lang="ru">
+        Больше информации на нашем сайте
+        <a href="https://organicmaps.app/ru/">organicmaps.app</a>
```
Comment (biodranik) [inline templates/faq/embedded-faq.html:171]:
```suggestion
        <a href="https://organicmaps.app/ru/">organicmaps.app/ru/</a>
```

File: templates/faq/embedded-faq.html [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+    </h4>
+    <h4 lang="de">
+        Für weitere Informationen besuchen Sie bitte unsere Website
+        <a href="https://organicmaps.app/">organicmaps.app</a>
+    </h4>
+    <h4 lang="es">
+        Para más información, visite nuestro sitio web
+        <a href="https://organicmaps.app/">organicmaps.app</a>
```
Comment (biodranik) [inline templates/faq/embedded-faq.html:179]:
```suggestion
        <a href="https://organicmaps.app/es/">organicmaps.app/es/</a>
```

File: templates/faq/embedded-faq.html [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
+    </h4>
+    <h4 lang="fr">
+        Pour plus d'informations, veuillez visiter notre site web
+        <a href="https://organicmaps.app/">organicmaps.app</a>
```
Comment (biodranik) [inline templates/faq/embedded-faq.html:183]:
```suggestion
        <a href="https://organicmaps.app/fr/">organicmaps.app/fr/</a>
```

File: templates/faq/embedded-faq.html [hunk 5]
```diff
@@ -0,0 +1,252 @@
...
+    </h4>
+    <h4 lang="pt">
+        Para mais informações, visite o nosso sítio na web
+        <a href="https://organicmaps.app/">organicmaps.app</a>
```
Comment (biodranik) [inline templates/faq/embedded-faq.html:187]:
```suggestion
        <a href="https://organicmaps.app/pt/">organicmaps.app/pt/</a>
```

File: templates/faq/embedded-faq.html [hunk 6]
```diff
@@ -0,0 +1,252 @@
...
+    </h4>
+    <h4 lang="pt-BR">
+        Para mais informações, visite o nosso sítio web
+        <a href="https://organicmaps.app/">organicmaps.app</a>
```
Comment (biodranik) [inline templates/faq/embedded-faq.html:191]:
```suggestion
        <a href="https://organicmaps.app/pt-BR/">organicmaps.app/pt-BR/</a>
```

File: templates/faq/embedded-faq.html [hunk 7]
```diff
@@ -0,0 +1,252 @@
...
+    </h4>
+    <h4 lang="pl">
+        Aby uzyskać więcej informacji, odwiedź naszą stronę internetową
+        <a href="https://organicmaps.app/">organicmaps.app</a>.
```
Comment (biodranik) [inline templates/faq/embedded-faq.html:195]:
```suggestion
        <a href="https://organicmaps.app/pl/">organicmaps.app/pl/</a>.
```

File: templates/faq/embedded-faq.html [hunk 8]
```diff
@@ -0,0 +1,252 @@
...
+    </h4>
+    <h4 lang="tr">
+        Daha fazla bilgi için lütfen
+        <a href="https://organicmaps.app/tr/">organicmaps.app/tr/</a>
+        web sitemizi ziyaret edin.
+    </h4>
+    <h4 lang="uk">
+        Для отримання додаткової інформації, будь ласка, відвідайте наш веб-сайт
+        <a href="https://organicmaps.app/">organicmaps.app</a>
```
Comment (biodranik) [inline templates/faq/embedded-faq.html:204]:
```suggestion
        <a href="https://organicmaps.app/uk/">organicmaps.app/uk/</a>
```

File: templates/faq/embedded-faq.html [hunk 9]
```diff
@@ -0,0 +1,252 @@
...
+    </h4>
+    <h4 lang="zh">
+        要获取更多信息，请访问我们的网站
+        <a href="https://organicmaps.app/">organicmaps.app</a>
```
Comment (biodranik) [inline templates/faq/embedded-faq.html:208]:
```suggestion
        <a href="https://organicmaps.app/zh-Hans/">organicmaps.app/zh-Hans/</a>
```

File: templates/faq/embedded-faq.html [hunk 10]
```diff
@@ -0,0 +1,252 @@
...
+            {# Add translations #}
+            {% for lang in faq_languages %}
+                {% set translated_faq_page = faq_page_en.translations | filter(attribute="lang", value=lang) | first %}
+                <h3 lang="{{ lang | replace(from="-Hans", to="") }}">{{ translated_faq_page.title }}</h3>
+            {% endfor %}
+        </dt>
+
+        <dd lang="en">
+            {{ faq_page_en.content | safe }}
+        </dd>
```
Comment (biodranik) [inline templates/faq/embedded-faq.html:227]: Is it possible to have one dd section for all translations inside it? E.g. dd with a div+lang or p+lang inside?

File: templates/faq/embedded-faq.html [hunk 11]
```diff
@@ -0,0 +1,252 @@
...
+            {% set translated_faq_page = faq_page_en.translations | filter(attribute="lang", value=lang) | first %}
+            {% set translated_faq_page = get_page(path=translated_faq_page.path) %}
+            <dd lang="{{ lang | replace(from="-Hans", to="") }}">
+                {{ translated_faq_page.content | safe }}
+            </dd>
+        {% endfor %}
+    </dl>
+{% endfor %}
+
+<a href="#" lang="en">Go to Top</a>
```
Comment (biodranik) [inline templates/faq/embedded-faq.html:239]: Looks like adding these translations into config.toml would be more flexible.

**Review by biodranik — COMMENTED** [review]
Thanks a lot for this PR!

> there are no questions "How to create a route and start navigation" and "I can't hear voice instructions"

Why? What is needed to fix it?

File: content/faq/embedded-faq.md [hunk 12]
```diff
@@ -0,0 +1,7 @@
+---
+title: Multilang single page FAQ
```
[the remark this answers is not in the document]
Comment (strump) [reply content/faq/embedded-faq.md:2]: This page has the same content as other FAQ pages. I think we don't need to index it by sesarch engines.

File: templates/faq/embedded-faq.html [hunk 13]
```diff
@@ -0,0 +1,252 @@
...
+            var elems = document.querySelectorAll('[lang="' + lang + '"]');
+            // Make them visible.
+            for (var i = 0; i < elems.length; ++i) elems[i].style.display = "block";
+        }
+    </script>
+</head>
+
+<body onload="showLanguage()">
+{# List of questions #}
+{% set faq_questions_en = [
```
[the remark this answers is not in the document]
Comment (strump) [reply templates/faq/embedded-faq.html:123]: We export only subset of FAQ questions. How can we tell which pages to include? Does ordering matters?

File: templates/faq/embedded-faq.html (hunk 1 above)
Comment (strump) [reply templates/faq/embedded-faq.html]: I used for-loop to generate links in format `organicmaps.app/{{ lang }}/`

File: templates/faq/embedded-faq.html (hunk 2 above)
Comment (strump) [reply templates/faq/embedded-faq.html]: I used for-loop to generate links in format `organicmaps.app/{{ lang }}/`

File: templates/faq/embedded-faq.html (hunk 3 above)
Comment (strump) [reply templates/faq/embedded-faq.html]: I used for-loop to generate links in format `organicmaps.app/{{ lang }}/`

File: templates/faq/embedded-faq.html (hunk 4 above)
Comment (strump) [reply templates/faq/embedded-faq.html]: I used for-loop to generate links in format `organicmaps.app/{{ lang }}/`

File: templates/faq/embedded-faq.html (hunk 5 above)
Comment (strump) [reply templates/faq/embedded-faq.html:170]: I used for-loop to generate links in format `organicmaps.app/{{ lang }}/`

File: templates/faq/embedded-faq.html (hunk 6 above)
Comment (strump) [reply templates/faq/embedded-faq.html]: I used for-loop to generate links in format `organicmaps.app/{{ lang }}/`

File: templates/faq/embedded-faq.html (hunk 7 above)
Comment (strump) [reply templates/faq/embedded-faq.html]: I used for-loop to generate links in format `organicmaps.app/{{ lang }}/`

File: templates/faq/embedded-faq.html (hunk 8 above)
Comment (strump) [reply templates/faq/embedded-faq.html]: I used for-loop to generate links in format `organicmaps.app/{{ lang }}/`

File: templates/faq/embedded-faq.html (hunk 9 above)
Comment (strump) [reply templates/faq/embedded-faq.html]: I used for-loop to generate links in format `organicmaps.app/{{ lang }}/`

File: templates/faq/embedded-faq.html (hunk 11 above)
Comment (strump) [reply templates/faq/embedded-faq.html]: Added new translation key 'go-to-top'.

File: content/faq/embedded-faq.md (hunk 12 above)
[the remark this answers is not in the document]
Comment (biodranik) [reply content/faq/embedded-faq.md:2]: Does it have `<meta name="robots" content="noindex" />` in the template?

**Comment by biodranik** [comment]
@strump can you please double-check if it's ready to merge and if the produced html works on iOS and Android?

**Comment by strump** [comment]
@biodranik ready to merge

**Review by biodranik — APPROVED** [review]
Thanks!

**Pull request merged**